client/src/legacy/TinyMCE_sslink.js

```diff
@@ -35,6 +35,9 @@ const plugin = {
 
     // Callback for opening the edit link dialog form
     function openLinkDialog() {
+      // Find "a" node, issue https://github.com/silverstripe/silverstripe-cms/issues/2439
```

Comment:
```suggestion
      // Find "a" node (we might have clicked on a child element e.g. "span")
```
There's no need to list the issue - git blame will inevitably lead us back to this PR and then to that issue if we really need that level of context on this.

Reply:
Done updated